src/constants/url.js

```diff
@@ -0,0 +1,3 @@
+const url = "https://ecommerce-champions.onrender.com";
```

Comment:
@sostene12 why do we still have this hard-coded value? I recommend you to rename the `url.js` to some thing like `keys.js` and then declare the `API_URL` there.

Hint:
```javascript
const keys = {
  API_URL: import.meta.VITE_BASE_URL
}
export default keys;
```